src/wallet/rpcwallet.cpp

```diff
@@ -2823,8 +2827,17 @@ UniValue bumpfee(const JSONRPCRequest& request)
         // new fee rate must be at least old rate + minimum incremental relay rate
         if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK()) {
             nNewFeeRate = CFeeRate(nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK());
+            if (specifiedConfirmTarget || payTxFee.GetFeePerK() == 0) {
```

Comment:
Actually, on further thought, we might as well have this apply even if payTxFee is set...  There's no philosophical difference in my mind between being willing to exceed it by `incrementalRelayFee` vs. `walletIncrementalRelayFee`.